src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_controlnet.py

```diff
@@ -594,6 +700,7 @@ def __call__(
         callback_steps: int = 1,
         cross_attention_kwargs: Optional[Dict[str, Any]] = None,
         controlnet_conditioning_scale: float = 1.0,
+        controlnet_conditions: Optional[List[ControlNetCondition]] = None,
```

Comment:
```suggestion
```

Would it be ok for you to just use the controlnet_conditions for internal usage of the pipeline? 

I'm a bit worried about exposing a very new design module such as `ControlNetCondition` to the main `__init__.py` and also think the user shouldn't have to learn about a new concept when using multi controlnet, this just passing multiple images and controlnet_conditioning_scale should be good enough :-)

Reply:
Ok. The benefits of ControlNetCondition may become apparent as more parameter extensions are added, but for now I'll keep it for internal use only and not expose it.

Reply:
Fixed in https://github.com/huggingface/diffusers/pull/2627/commits/d1acef4b7c372f9a1c77f526b0d67f727b81064d